contest/contest_info.py

```diff
@@ -199,12 +206,16 @@ def can_register_return_status(user, contest):
         return IS_ADMIN
 
     return OK
-
+'''
+return boolean
+'''
 def can_register(user, contest):
     if can_register_return_status(user, contest) is OK:
         return True
     return False
-
+'''
+depend on error msg and write to log
+'''
 def can_register_log(user, contest):
```

Comment:
As you said, a checking should not write log.
I think this function should be removed.